tracee-ebpf/tracee/events_processor.go

```diff
@@ -216,16 +217,17 @@ func (t *Tracee) processEvent(event *external.Event) error {
 			return err
 		}
 	case SchedProcessExitEventID:
-		delete(t.processTree.processTreeMap, event.HostProcessID)
+		time.Sleep(time.Second)
```

Comment:
Please don't do that.
It blocks the processor thread for one second, thus no other events can be processed during this time.